internal/v1/api.yaml

```diff
@@ -718,6 +752,26 @@ components:
         - fedora-38
         - fedora-39
         - fedora-40
+    UpdateBlueprintRequest:
+      type: object
+      additionalProperties: false
+      required:
+        - distribution
+        - image_requests
+        - customizations
+      properties:
+        distribution:
+          $ref: '#/components/schemas/Distributions'
+        image_requests:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/ImageRequest'
+          uniqueItems: true
+          description: |
+            Array of image requests. Having more image requests in a single blueprint is currently not supported.
+        customizations:
+          $ref: '#/components/schemas/Customizations'
```

Comment:
So you removed "name" and "description".

Isn’t simply allowing update of all the fields more straightforward for the API user? I always try to find solutions which are easier for the user, not for the developer. This would mean updating both tables in a transaction and documenting the behavior: name and description are not subject of versioning. @croissanne ?

If we do it this way, then we can actually reuse the OpenAPI component between create and update.

Reply:
Yeah, updating the description and the name makes totally sense to me as well. I will incorporate the idea into the PR, and we can decide later, if we want to drop this, what do you think?

Reply:
Thanks, I see you added it into the code logic, can you reuse the OpenAPI type used for creation here?